Week3/homework/step3-bonus.js

```diff
@@ -1,12 +1,10 @@
 'use strict';
 
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
-
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const uniqValArr = arr.filter((el, i, array) => array.indexOf(el) === i);
```

Comment:
Just a suggestion here - try and read about a new concept in ES6 which is called Sets and see if you can use it to make this code shorter. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set